### PR TITLE
display all learning reasourse offered bys

### PR DESCRIPTION
**Pull request by abeglova**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
Screen Shot 2019-12-26 at 6 57 03 PM
Screen Shot 2019-12-26 at 6 57 36 PM

- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2500

#### What's this PR do?
This pr updates the ui to display all the offered by values if there are multiple

#### How should this be manually tested?
Find or create a learning resource with multiple offered by values. Verify that all the offered by values are displayed in the UI

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)

### CODE REVIEW

File: static/js/components/ExpandedLearningResourceDisplay.js [hunk 1]
```diff
@@ -109,7 +108,7 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
   const url = selectedRun && selectedRun.url ? selectedRun.url : object.url
   const cost = selectedRun ? minPrice(selectedRun.prices, true) : null
 
-  const offeredBy = getPreferredOfferedBy(object.offered_by)
+  const offeredBy = object.offered_by.join(", ")
```
Comment (mbertrand) [inline static/js/components/ExpandedLearningResourceDisplay.js]: Maybe something to check with @pdpinch or @abdulkdawson about - the button to take a course on the drawer display now reads "Take course on MITx, Micromasters" for example if it has two offered_by values, but the link goes to MITx.  Is this the desired behavior?  

Screen Shot 2019-12-30 at 11 51 37 AM

**Comment by pdpinch** [comment]
Maybe I’m confused, but I thought it was supposed to be “Take Course on {platform}” not “Take Course on {offered_by}"

> Maybe something to check with @pdpinch <https://github.com/pdpinch> or @abdulkdawson <https://github.com/abdulkdawson> about - the button to take a course on the drawer display now reads "Take course on MITx, Micromasters" for example if it has two offered_by values, but the link goes to MITx. Is this the desired behavior?

**Comment by abdulkdawson** [comment]
@mbertrand @pdpinch The button should only say "Take Course" with the platform icon to the right of the button.

**Comment by mbertrand** [comment]
Thanks @abdulkdawson.  That change should be handled by another open PR #2513, so I think it should be okay to merge this as is, since it's not visible on production yet anyway.

**Pull request merged**